vms/platformvm/txs/executor/reward_validator_test.go

```diff
@@ -340,9 +344,360 @@ func TestRewardDelegatorTxExecuteOnCommit(t *testing.T) {
 	require.Equal(env.config.MinValidatorStake, vdrSet.GetWeight(vdrNodeID))
 }
 
+func TestRewardDelegatorTxExecuteOnCommitPostDelegateeDeferral(t *testing.T) {
```

Comment:
Can we add some tests for the `OnAbortState`? Either inside these tests or as separate tests.

Reply:
[252652d](https://github.com/ava-labs/avalanchego/pull/1262/commits/252652d2ef609be096f66dfeffaf48479df58e01)